caps/types.go

```diff
@@ -53,6 +54,13 @@ func (t *BoolFileType) Name() string {
 	return "bool-file"
 }
 
+var boolFileAllowedPathPatterns = []*regexp.Regexp{
+	// The brightness of standard LED class device
+	regexp.MustCompile("^/sys/class/leds/[^/]+/brightness$"),
```

Comment:
I'd like to keep it for two reasons:
 - it is useful for common keyboard LEDs (caps lock, scroll lock, etc) and for testing on x86
 - vast majority of LEDs have only two states 0 and !0  (no PWM support)

On a given dell laptop, the gadget snap can suppress that LED capability if it doesn't behave correctly. In every other case LEDs are a valid _candidate_ and thus should be allowed.